pkg/operation/common/types.go

```diff
@@ -404,6 +404,21 @@ const (
 	// ETCDImageName is the name of the ETCD image.
 	ETCDImageName = "etcd"
 
+	// ETCDBackupRestoreImageName is the name of the ETCD backup-restore image.
+	ETCDBackupRestoreImageName = "etcd-backup-restore"
+
+	// EtcdDruidImageName is the name of Etcd Druid image
+	EtcdDruidImageName = "etcd-druid"
+
+	// CSINodeDriverRegistrarImageName is the name of driver registrar - https://github.com/kubernetes-csi/node-driver-registrar
+	CSINodeDriverRegistrarImageName = "csi-node-driver-registrar"
```

Comment:
Can we please remove these CSI constants again?

Reply:
Done